frontend/public/components/utils/units.js

```diff
@@ -379,6 +379,10 @@ export const convertToBaseValue = (value) => {
     return units.dehumanize(value, 'decimalBytesWithoutB').value;
   }
 
+  if (TYPES.binaryBytes.units.includes(unit)) {
```

Comment:
What value are we parsing the has the `B` on the end? This utility is meant to parse the values returned from the API, which should not have the trailing `B`. I'm not sure we should try to handle those cases since they aren't valid.